[docs] added details about import data file with S3 (#17322)
yugabyte/yugabyte-db: docs/content/preview/migrate/reference/yb-voyager-cli.md

| Argument | Description/valid options |
| :------- | :------------------------ |
| [--batch-size](#batch-size) <number> | Size of batches generated for ingestion during [import data]. |

> | [--data-dir](#data-dir) <path> | Path to the directory/S3 prefixed bucket URI containing the data files to import. |

Reviewer: ```suggestion
| [--data-dir](#data-dir) <path> | Path to the directory or S3-prefixed URI of the bucket containing the data files to import. |
```

--data-dir s3://voyager-data
```

> The authentication mechanism used when accessing the S3 bucket through yb-voyager is the same as using the AWS CLI tool to access the bucket.

Reviewer: ```suggestion
The authentication mechanism for accessing an S3 bucket using yb-voyager is the same as that used by the AWS CLI.
```
